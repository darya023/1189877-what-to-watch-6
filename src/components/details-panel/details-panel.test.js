import React from 'react';
import {render} from '@testing-library/react';
import DetailsPanel from './details-panel';

it(`Should DetailsPanel render correctly`, () => {
  const {container} = render(<DetailsPanel
    genre="test"
    year={2000}
    director="Name"
    starring={[`Name Test`, `Name Test2`]}
    duration={150}
  />);
  expect(container).toMatchSnapshot();
});
