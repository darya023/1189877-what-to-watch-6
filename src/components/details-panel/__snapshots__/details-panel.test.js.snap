// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should DetailsPanel render correctly 1`] = `
<div>
  <div
    class="movie-card__text movie-card__row"
  >
    <div
      class="movie-card__text-col"
    >
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Director
        </strong>
        <span
          class="movie-card__details-value"
        >
          Name
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Starring
        </strong>
        <span
          class="movie-card__details-value"
        >
          Name Test
          <br />
          Name Test2
          <br />
        </span>
      </p>
    </div>
    <div
      class="movie-card__text-col"
    >
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Run Time
        </strong>
        <span
          class="movie-card__details-value"
        >
          2h 30m
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Genre
        </strong>
        <span
          class="movie-card__details-value"
        >
          test
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Released
        </strong>
        <span
          class="movie-card__details-value"
        >
          2000
        </span>
      </p>
    </div>
  </div>
</div>
`;
